.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,35 @@
+<!--
```

Comment:
I think we don't want the license headers in templates file(?)

Reply:
👍 I will update this once #7217 is completed and merged.